Controller/RegistrationFOSUser1Controller.php

```diff
@@ -56,10 +56,10 @@ public function registerAction()
                 $url = $this->generateUrl('fos_user_registration_check_email');
             } else {
                 $authUser = true;
-                $route = $this->get('session')->get('sonata_basket_delivery_redirect');
+                $route = $this->get('session')->get('sonata_user_redirect_url');
 
                 if (null !== $route) {
-                    $this->get('session')->remove('sonata_basket_delivery_redirect');
+                    $this->get('session')->remove('sonata_user_redirect_url');
```

Comment:
Hi @greg0ire,

Indeed, this was for the `ecommerce` bundle suite in order to let user come back on its basket when he choose to register from the basket page (https://github.com/sonata-project/ecommerce/blob/7e778d57d1229df3f73b453404f9e7360b0ca923/src/BasketBundle/Controller/BasketController.php#L53).